account_statement_import_file/wizard/account_statement_import.py

```diff
@@ -335,9 +335,10 @@ def _create_bank_statements(self, stmts_vals, result):
                         vals["sequence"] = seq
                 # Remove values that won't be used to create records
                 st_vals.pop("transactions", None)
+                context = st_vals.pop("context", {})
```

Comment:
Be a bit more specific:

```suggestion
                context = st_vals.pop("creation_context", {})
```

Reply:
:+1: